fsw/inc/ds_platform_cfg.h

```diff
@@ -441,10 +441,10 @@
  *
  *  \par Limits
  *       This parameter must be set to one of the following:
- *       true  = add move pathname field to Destination File Table
- *       false = do not add move pathname to Destination File Table
+ *       non-empty string = add move pathname field to Destination File Table
+ *       empty string = do not add move pathname to Destination File Table
  */
-#define DS_MOVE_FILES true
+#define DS_MOVE_FILES "Move to downlink directory"
```

Comment:
Would recommend changing this macro name.  DS_MOVE_FILES should be gone entirely, and a new one line `DS_MOVEFILES_DESTDIR` exists instead